Doc/library/importlib.metadata.rst

```diff
@@ -173,10 +173,10 @@ for more information on entry points, their definition, and usage.
 
 *Compatibility Note*
 
-The "selectable" entry points were introduced in ``importlib_metadata``
+The "selectable" entry points were introduced in ``importlib.metadata``
 3.6 and Python 3.10. Prior to those changes, ``entry_points`` accepted
 no parameters and always returned a dictionary of entry points, keyed
-by group. With ``importlib_metadata`` 5.0 and Python 3.12,
+by group. With ``importlib.metadata`` 5.0 and Python 3.12,
```

Comment:
I don't think these changes are correct, as they refer to the [`importlib_metadata`](https://pypi.org/project/importlib-metadata/) package.

Reply:
I'll change these two back to `importlib_metadata` - thanks